cumulusci/tasks/marketing_cloud/deploy.py

```diff
@@ -28,9 +30,19 @@ class MarketingCloudDeployTask(BaseMarketingCloudTask):
         "package_zip_file": {
             "description": "Path to the package zipfile that will be deployed.",
             "required": True,
-        }
+        },
+        "custom_inputs": {
+            "description": "Specify custom inputs to the deployment task. Takes a mapping from input key to input value (e.g. 'companyName:Acme,companyWebsite:https://www.salesforce.org:8080').",
+            "required": False,
+        },
     }
 
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+        self.custom_inputs = process_list_of_pairs_dict_arg(
+            self.options["custom_inputs"]
```

Comment:
This needs to be 
```suggestion
            self.options.get("custom_inputs")
```

so it doesn't break when the option isn't specified.